.github/workflows/ci.yml

```diff
@@ -68,6 +68,7 @@ jobs:
             flag-for-pinia: '--pinia'
             flag-for-vitest: '--vitest'
             flag-for-e2e: '--cypress'
+            flag-for-vue-devtools: 'vue-devtools'
```

Comment:
FYI, items above all have a `--` prefix. This one doesn't. Not sure if that has any impact.